src/monster.cpp

```diff
@@ -932,7 +926,7 @@ void Monster::onThinkDefense(uint32_t interval)
 		}
 	}
 
-	if (!isSummon() && summons.size() < mType->maxSummons) {
+	if (!isSummon() && summons.size() < mType->maxSummons && hasFollowPath) {
```

Comment:
@marksamman 
I would love some feedback, so i can make the necessary changes, if there is needed.